app/components/quiz/QuizAnswer.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react';
+import AppStore from 'store/appStore';
+import 'quiz.scss';
+import jquery from 'jquery';
 
 export default React.createClass({
   displayName: 'QuizAnswer',
 
   propTypes: {
-    answer: React.PropTypes.object.isRequired
+    answer: React.PropTypes.object.isRequired,
+    isSelected: React.PropTypes.object.isRequired,
+    id: React.PropTypes.object.isRequired,
+    colors: React.PropTypes.string.isRequired,
+    answerClicked: React.PropTypes.func.isRequired,
+    type: React.PropTypes.string.isRequired
+  },
+
+  onClick: function(e) {
+    // it is clicked
+    var $ = jquery;
+    var answer = this.props.answer;
+    var id = '#' + this.props.id + answer.question;
+    if (this.props.colors !== null && !this.props.isSelected) {
+      $(id).css('background-color', this.props.colors);
```

Comment:
instead of mutating the DOM nodes directly, you should `setState` to cause a `render` with new color state